app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
```

Comment:
- [ ] OPTIONAL
`Like` model should have method that updates the likes counter for a post. This should be similar to the example below:

```ruby
# app/models/like.rb
...

def update_likes_counter
  post.increment!(:likes_counter)
end
```

Reply:
Hi @rloterh,
Many thanks for your comment.
But I already applied this requirement using (counter_cache: :likes_counter) which should be more efficient.
Do you think I need to change that?

Reply:
Hey @jaferIdrees 
That's good implementation from you! Cheers! 😃